Agreements/Internal/C_MAE.txt

```diff
@@ -19,4 +19,7 @@ COPX:*:*:EVSON:EGCN:*:London AC Daventry:Scottish PC East:*:20000:EVSON40
 
 ;fixme POL45 or TNT also
 COPX:*:*:BETAX:EGNT:*:London AC Daventry:Scottish PC East:*:28000:BETAX
-COPX:*:*:BETAX:EGNV:*:London AC Daventry:Scottish PC East:*:28000:BETAX
\ No newline at end of file
+COPX:*:*:BETAX:EGNV:*:London AC Daventry:Scottish PC East:*:28000:BETAX
+
+COPX:*:*:MAMUL25:EGNT:*:London AC Daventry:Scottish PC East:*:28000:MAMUL25
```

Comment:
BETAX is 28 miles from MAMUL - so we can keep this as it is, delete new lines 24/25

Reply:
Allright makes sence